cmd/cvdr/main.go

```diff
@@ -127,6 +129,11 @@ func main() {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(2)
 	}
+	commandLine := strings.Join(os.Args, " ")
+	if err := metrics.SendLaunchCommand(commandLine); err != nil {
```

Comment:
This seems to be sending the information, without the option to turn it off. User consent should be obtained before sending any metrics information to the servers, at least on the first execution.

Reply:
I created this bug b/309513939 to follow the necessary changes for metrics collection flag in the next CLs.